Code/GraphMol/CIPLabeler/catch_tests.cpp

```diff
@@ -1075,3 +1075,97 @@ M  END
+   16.0673  -10.4532    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   13.4693  -10.4532    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   13.4693   -8.9532    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   12.1702   -8.2032    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   10.8712   -8.9532    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   10.8712  -10.4532    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   12.1702  -11.2030    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   14.7682   -8.2030    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   12.1702  -12.7030    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   10.8711  -13.4530    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   13.4692  -13.4530    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  2  0  0  0  0
+  2  3  1  0  0  0  0
+  3  4  2  0  0  0  0
+  4  5  1  0  0  0  0
+  5  6  2  0  0  0  0
+  6  1  1  0  0  0  0
+  1  7  1  0  0  0  0
+  7  8  2  0  0  0  0
+  8  9  1  0  0  0  0
+  9 10  2  0  0  0  0
+ 10 11  1  0  0  0  0
+ 11 12  2  0  0  0  0
+  7 12  1  1  0  0  0
+  8 13  1  0  0  0  0
+ 12 14  1  0  0  0  0
+ 14 15  1  0  0  0  0
+ 14 16  1  0  0  0  0
+M  END
+)";
+
+    std::string molBlock2 = R"(
+Generated by WebMolKit
+
+ 16 17  0  0  1  0  0  0  0  0999 V2000
+   11.5862  -13.1456    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   14.1816  -13.1449    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   12.8863  -12.3955    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   14.1816  -14.6463    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   11.5862  -14.6530    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
+   12.8896  -15.3956    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   10.2872  -12.3954    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    7.6918  -12.3979    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    8.9876  -13.1465    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    7.6908  -10.8965    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   10.2862  -10.8880    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    8.9823  -10.1464    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    8.9876  -14.6465    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    7.6886  -15.3965    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   10.2867  -15.3964    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   11.5826  -10.1334    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+  3  1  2  0  0  0  0
+  1  5  1  0  0  0  0
+  5  6  2  0  0  0  0
+  6  4  1  0  0  0  0
+  4  2  2  0  0  0  0
+  2  3  1  0  0  0  0
+  1  7  1  0  0  0  0
+  7 11  2  0  0  0  0
+ 11 12  1  0  0  0  0
+ 12 10  2  0  0  0  0
+ 10  8  1  0  0  0  0
+  8  9  2  0  0  0  0
+  9 13  1  0  0  0  0
+ 13 14  1  0  0  0  0
+ 13 15  1  0  0  0  0
+ 11 16  1  0  0  0  0
+  7  9  1  1  0  0  0
+M  END
+)";
+
+    UseLegacyStereoPerceptionFixture useLegacy(false);
+    auto found = cipLabels(molBlock);
+
+    CHECK(found ==
```

Comment:
This style of testing made it through my review of the giant atropisomer PR, but I would rather not continue using it. It's just too hard as a human to figure out what is actually being looked for. Also, if/when one of these tests fails, it's going to be a pain to figure out what actually changed based on the long stream of commas and spaces.

Here and in future tests, please be explicit about what you are looking for, i.e. use something like:
```
CHECK(mol->getBondWithIdx(n)->getProp<std::string>("_CIPCode")=="P");
```

Reply:
I also think it would be worthwhile to switch the wedging on the single bond here and confirm that you get "M" instead of "P"

Reply:
I made the changes as suggested.